microprofile/telemetry/src/main/java/io/helidon/microprofile/telemetry/TelemetryCdiExtension.java

```diff
@@ -28,8 +28,15 @@
  */
 public class TelemetryCdiExtension implements Extension {
 
+    /**
+     * Property name indicating the presence of the OpenTelemetry Java agent.
+     */
+    static final String OTEL_AGENT_PRESENT = "otel.agent.present";
```

Comment:
See above - I think this is already built in Helidon (`io.helidon.tracing.providers.opentelemetry.HelidonOpenTelemetry.AgentDetector`)